packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-payment-initialize-options.ts

```diff
@@ -86,25 +86,14 @@ export default interface PayPalCommerceCreditCardsPaymentInitializeOptions {
      * The callback that gets called when there is an issue with rendering credit card fields
      */
     onCreditCardFieldsRenderingError?: (error: unknown) => void;
-}
 
-// TODO: this interface should be removed
-// We should give some time for other developers to update their PPC CC components
-// with paypalcommercecreditcard.form instead of paypalcommerce.form.
-// The interface can be removed after 14.06.2023
-export interface DeprecatedPayPalCommerceCreditCardsPaymentInitializeOptions {
     /**
-     * The form is data for Credit Card Form
+     * The callback that gets called when PayPal SDK load complete
      */
-    form?: HostedFormOptions;
-
-    /**
-     * The callback that gets called when there is an issue with rendering credit card fields
-     */
-    onCreditCardFieldsRenderingError?: (error: unknown) => void;
+    onLoadComplete?: () => void;
 }
 
 export interface WithPayPalCommerceCreditCardsPaymentInitializeOptions {
     paypalcommercecreditcards?: PayPalCommerceCreditCardsPaymentInitializeOptions;
-    paypalcommerce?: DeprecatedPayPalCommerceCreditCardsPaymentInitializeOptions; // FIXME: the option is deprecated
+    paypalcommerce?: PayPalCommerceCreditCardsPaymentInitializeOptions;
```

Comment:
Please, add a comment that the paypalcommerce option is deprecated for current strategy